src/app/loadout-builder/utils.ts

```diff
@@ -401,3 +409,13 @@ export function someModHasEnergyRequirement(
 ) {
   return mods.some((mod) => mod.mod.plug.energyCost!.energyType !== DestinyEnergyType.Any);
 }
+
+export const Armor2ModPlugCategoriesTitles = {
+  [ModPickerCategories.general]: t('LB.General'),
+  [ModPickerCategories.helmet]: t('LB.Helmet'),
+  [ModPickerCategories.gauntlets]: t('LB.Gauntlets'),
+  [ModPickerCategories.chest]: t('LB.Chest'),
+  [ModPickerCategories.leg]: t('LB.Legs'),
+  [ModPickerCategories.classitem]: t('LB.ClassItem'),
+  [ModPickerCategories.seasonal]: t('LB.Seasonal'),
```

Comment:
These can't be translated here - `t` reads the current translation context which both isn't always defined at the point where this is loaded, and can change if the user changes their language. These should use `tl` and be translated at the point where they're rendered. The export also shouldn't use TitleCase - that should be reserved for component, class, or interface names.

Reply:
Good point, I didn't think of the translation context. I will just have to duplicate it.